archetypes/adr.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "{{ replace .Name "-" " " | title }}"
+type: "adrs"
```

Comment:
I think you forgot to remove this based on your comment.

Reply:
Added now.